### ISSUE TITLE
Missing aria-label for toolbox

### ISSUE THREAD

**Issue by microbit-matt-hillsdon** [RaspberryPiFoundation/blockly#9292]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Description

This is based on reviewing https://google.github.io/blockly-keyboard-experimentation/screenreader/

The toolbox has role=tree but no aria label. As a result when you move to it you don't know that it's the toolbox. Try moving left/right between the toolbox and flyout with keyboard navigation.

NVDA reads: "tree view, Logic 1 of 8 level 1"

For the flyout it terms it "Flyout tree view" which appears to be based on the aria label.

This seems like a simple fix that will improve output.

### Reproduction steps

1. Use NVDA on the keyboard nav demo to read as you move focus between toolbox and flyout 
2. Note you can understand when you move to the flyout but not when you move to the toolbox
